setup.cfg

```
@@ -1,39 +1,40 @@
-[metadata]
+ [metadata]
```

The additional space might be a problem. 
```suggestion
[metadata]
```